src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -70,6 +70,7 @@
     <TSqlScript Include="Features\Schema\Sql\Tables\*.sql" />
     <TSqlScript Include="Features\Schema\Sql\Types\*.sql" />
     <None Remove="Features\Schema\Migrations\12.diff.sql" />
+    <None Remove="Features\Schema\Migrations\13.diff.sql" />
```

Comment:
shouldn't this line and above be handled by a wildcard?

Reply:
Yes, VS adds automatically. Removed it